site/src/_pages/docs/0_index.mdx

```diff
@@ -9,6 +9,20 @@ The following pages provide a practical introduction for developers who want to:
 
 For the full technical details, read [the Block Protocol specification](https://blockprotocol.org/spec).
 
+### Why is structured data useful?
+
+Structured data allows computers to understand information. It is easier to create and easier to understand than unstructured data.
```

Comment:
"It is easier to create and easier to understand than unstructured data." I take issue with the claim that it's easier to create. The Block Protocol was (partly) born of frustration that this wasn't the case. When in rapid-notetaking mode most people just want to dump ideas onto a page and not worry about structure. But blocks that are both useful to folks in that type of a setting (where speed matters), and which organically add semantic meaning to the information being captured at the same time, lay the groundwork for this to be true.

Reply:
I was unsure about that claim too - copied off the rough notes from Joel, but figured it was in the context of pre-selecting entities from a known list vs. relying on memory. We can adapt to "Structured data makes it much easier for computers to understand information"

Reply:
Updated to: "Structured data makes it much easier for computers to understand the information we give them."